### ISSUE TITLE
Lab 4 Note Regarding ShareACab

### ISSUE THREAD

**Issue by eLTehh** [nus-cs2030/2223-s1#84]
Not actually a problem I'm having, but the question was unclear, so:
Dividing the fee by number of passengers comes AFTER the peak hour surcharge. Hope this helps anyone having problems with calculations.

Would also be nice if this was made clearer in the question!

**Comment by Wenbo-J** [comment]
Yeah ikr! I also felt that this part of the question could be clearer

**Comment by wjiayis** [comment]
Thanks man!

**Comment by danqiye1** [comment]
Prof Henry has updated the lab4 writeup. Thanks for the feedback.
